source/renderer/app/components/staking/delegation-setup-wizard/DelegationSetupWizardDialog.js

```diff
@@ -45,6 +45,25 @@ type Props = {
   isTrezor: boolean,
 };
 
+const getOversaturationPercentage = (
+  selectedWallet: ?Wallet,
+  selectedPool: ?StakePool,
+  maxDelegationFunds: number
+): number => {
+  if (
+    !selectedPool ||
+    !selectedWallet ||
+    (selectedWallet.delegatedStakePoolId ||
```

Comment:
@mchappell are we sure this is the right logic?

e.g. we have a case: `StakePoolA` is the `selectedPool`:

```
selectedWallet.delegatedStakePoolId = StakePoolB
selectedWallet.lastDelegatedStakePoolId = StakePoolA
```

☝️  this case would result in **not showing** the oversaturation notification which is fine 


```
selectedWallet.delegatedStakePoolId = StakePoolA
selectedWallet.lastDelegatedStakePoolId = StakePoolB
```

☝️  this case would result in **not showing** the oversaturation notification and it should!